internal/logger/targets.go

```diff
@@ -108,3 +111,37 @@ func AddTarget(t Target) error {
 
 	return nil
 }
+
+func cancelAllTargets() {
+	for _, tgt := range Targets() {
+		tgt.Cancel()
+	}
+}
+
+func initTargets(cfg Config) (tgts []Target, err error) {
+	for _, l := range cfg.HTTP {
+		if l.Enabled {
+			t := http.New(l)
+			if err = t.Init(); err != nil {
+				return tgts, err
+			}
+			tgts = append(tgts, t)
+		}
+	}
+	return tgts, err
+}
+
+// UpdateTargets swaps targets with newly loaded ones from the cfg
+func UpdateTargets(cfg Config) error {
+	cancelAllTargets() // cancel running targets
```

Comment:
Having it here, will drop messages from when this starts until `targets` has been updated.

Reply:
Thanks, fixed.